Livecheckables/chkrootkit.rb

```diff
@@ -1,6 +1,6 @@
 class Chkrootkit
   livecheck do
     url :homepage
-    regex(%r{href="/download/">chkrootkit ([0-9,.]+)})
+    regex(/href=.*?download[^>]+>chkrootkit v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/href=.*?download[^>]*>chkrootkit v?(\d+(?:\.\d+)+)/i)
```

Making this optional would allow for `href=download>`, which may happen if an upstream website doesn't use `"` or `'` around the `href` attribute (which I've unfortunately seen before).